src/components/WebCLI/WebCLI.tsx

```diff
@@ -20,6 +20,8 @@ import "./_webcli.scss";
 export enum Label {
   CONNECTION_ERROR = "Unable to connect to the model.",
   AUTHENTICATION_ERROR = "Unable to authenticate.",
+  NOT_OPEN_ERROR = "WebSocket connection is not open.",
```

Comment:
Maybe this should say something like "Lost connection to model" or something similar.